code/game/gamemodes/cult/runes.dm

```diff
@@ -566,6 +570,10 @@ var/list/teleport_other_runes = list()
 		R.visible_message("<span class='danger'>[R] fades away.</span>")
 		R.invisibility = INVISIBILITY_OBSERVER
 		R.alpha = 100 //To help ghosts distinguish hidden runes
+	for(var/mob/dead/observer/O in orange(3,src))
+		if(!O.invisibility)
+			O << "<span class'deadsay'><i>You suddenly feel as if you've vanished...</i></span>"

```

Comment:
What do you mean? The ghosts' invisibility is set to their normal invisibility. (I just fixed an issue where I forgot to change the variable, if that was the issue than it's fixed.)